tests/e2e/conftest.py

```diff
@@ -26,12 +64,6 @@ def pytest_addoption(parser):
         default=None,
         help="Ids of questions to be validated. Check json file for valid ids.",
     )
-    parser.addoption(
-        "--eval_threshold",
-        type=float,
-        default=None,
-        help="Threshold value to be used for similarity score.",
-    )
```

Comment:
not sure where this change came from but i don't think it should be part of this PR